src/common/ceph_strings.cc

```diff
@@ -153,9 +153,11 @@ uint64_t ceph_release_features(int r)
 		return req;
 
 	req |= CEPH_FEATUREMASK_CRUSH_CHOOSE_ARGS; // and overlaps
-	if (r <= CEPH_RELEASE_LUMINOUS)
+	if (r <= CEPH_RELEASE_QUINCY)
 		return req;
 
+	req |= CEPH_FEATUREMASK_SERVER_REEF; // upmap-primary
```

Comment:
Nit: I'd add
```
if (r <= CEPH_RELEASE_REEF)
        return req;
```
here to preserve the existing "structure" of this method.  Otherwise one might get the impression that we aren't done with handling reef at this point yet.